assets/javascripts/discourse/controllers/s-show.js.es6

```diff
@@ -21,6 +23,11 @@ export default Controller.extend({
     bootbox.alert(I18n.t(`discourse_subscriptions.${path}`));
   },
 
+  @discourseComputed()
+  isAnonymous() {
```

Comment:
You could probably use Ember's `not` helper and `currentUser` from the current context. So, something like: 

```
isAnonymous: not("currentUser")
```